nva-datamodel-java/src/main/java/no/unit/nva/model/CristinIdentifier.java

```diff
@@ -0,0 +1,7 @@
+package no.unit.nva.model;
+
+public record CristinIdentifier(String sourceName, String value) implements AdditionalIdentifierBase {
+
+    static final String TYPE = "CristinIdentifier";
+
```

Comment:
What is the sourceName here? The result uri? If so, maybe URI

Reply:
suggest doing SourceName class for all the new identifiers. source is either cristin, brage and  probably ova